src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -661,6 +661,13 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         // Discover translation directories
         $dirs = array();
+
+        foreach ($config['paths'] as $path) {
+            if (is_dir($path)) {
+                $dirs[] = $path;
+            }
```

Comment:
The `TranslatorPass` is executed before the optimization passes (in which parameters are resolved). To be on the safe side, the compiler pass must be registered as a `PassConfig::TYPE_BEFORE_REMOVING` pass.